region/maridia/outer/Mt. Everest.json

```diff
@@ -906,6 +844,35 @@
         "In the likely event of failure, the room will need to be reset and the crabs repositioned for the next attempt."
       ]
     },
+    {
+      "link": [2, 7],
```

Comment:
I think these [2, 7] G-mode strats may be pointless, since with Gravity alone you can just Gravity jump 2->7 (though that seems to be missing from the logic).